process/smartContract/process.go

```diff
@@ -750,6 +750,23 @@ func (sc *scProcessor) processVMOutput(
 		return nil, nil, process.ErrNilTransaction
 	}
 
+	if sc.isTooMuchGasPutInsideTx(vmOutput.GasRemaining, tx, acntSnd) {
+		log.Warn("scProcessor.processVMOutput: too much gas has been put inside tx",
```

Comment:
instead of log.Warn - which will fill the validators logs, I suggest to add a new ReturnMessage as well. Through that the user will get a notification in his wallet.

Reply:
Done